api/pom.xml

```diff
@@ -122,5 +122,16 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+	<dependency>
```

Comment:
Format appears a bit off. Not critical but we should attempt to follow a style guide for formatting.

Reply:
Fixed